### PR TITLE
Adding a complete how to of LokiStack on ROSA

### PR DESCRIPTION
**Pull request by andyrepton**

### CODE REVIEW

File: content/o11y/openshift-logging-lokistack/index.md [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+
+## Prerequisites
+
+1. OpenShift CLI (oc)
+1. Rights to install operators on the cluster
+1. Access to create S3 buckets (AWS/ROSA), Blob Storage Container (Azure), Storage Bucket (GCP)
+
+## For ROSA
+
+Create environment variables to use later in this process by running the following commands:
```
Comment (paulczar) [inline content/o11y/openshift-logging-lokistack/index.md:19]: 1.

File: content/o11y/openshift-logging-lokistack/index.md [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+
+```bash
+$ export REGION=$(oc get infrastructure cluster -o=jsonpath="{.status.platformStatus.aws.region}")
```
Comment (paulczar) [inline content/o11y/openshift-logging-lokistack/index.md:22]: all of these code blocks need to be indented, or they break the ordered list and it goes back to 1.

**Review by paulczar — CHANGES_REQUESTED** [review]

File: content/o11y/openshift-logging-lokistack/index.md [hunk 3]
```diff
@@ -0,0 +1,572 @@
+---
+date: '2021-07-27'
+title: Configuring OpenShift Logging using LokiStack on ROSA and (soon) ARO
+tags: ["Observability", "OCP"]
+authors:
+  - Andy Repton
+---
+
+A guide to shipping logs and metrics on OpenShift using the new LokiStack setup
```
Comment (bhardesty) [inline content/o11y/openshift-logging-lokistack/index.md:9]: Is the purpose of this obvious to users? If not, you might add a bit more here to explain why someone would/should do this.

File: content/o11y/openshift-logging-lokistack/index.md [hunk 4]
```diff
@@ -0,0 +1,572 @@
...
+
+![logging overview](./logging-overview.png)
+
+## Prerequisites
+
+1. OpenShift CLI (oc)
+1. Rights to install operators on the cluster
+1. Access to create S3 buckets (AWS/ROSA), Blob Storage Container (Azure), Storage Bucket (GCP)
+
+## For ROSA
```
Comment (bhardesty) [inline content/o11y/openshift-logging-lokistack/index.md:21]: I think the other Cloud Experts tutorials name this section "Setting up your environment", so I'd suggest that here as well.

File: content/o11y/openshift-logging-lokistack/index.md [hunk 5]
```diff
@@ -0,0 +1,572 @@
...
+        ```
+
+        Example Output
+
+        ```
+        NAME                     DISPLAY                     VERSION   REPLACES                 PHASE
+        loki-operator.v5.9.3     Loki Operator               5.9.3     loki-operator.v5.9.2     Succeeded
+        ```
+
+### If you are using OpenShift 4.14 or higher on AWS (ROSA)
```
Comment (bhardesty) [inline content/o11y/openshift-logging-lokistack/index.md:192]:
```suggestion
### Create the LokiStack installation (OpenShift 4.14 or higher on AWS (ROSA))
```

File: content/o11y/openshift-logging-lokistack/index.md [hunk 6]
```diff
@@ -0,0 +1,572 @@
...
+          name: logging-loki-aws 
+          type: s3 
+          credentialMode: token
+      storageClassName: gp3-csi
+      tenants:
+        mode: openshift-logging
+    EOF
+    ```
+
+### If you are using OpenShift 4.13 or lower on AWS (ROSA), or are using a Non-STS cluster
```
Comment (bhardesty) [inline content/o11y/openshift-logging-lokistack/index.md:229]:
```suggestion
### Create the LokiStack installation (OpenShift 4.13 or lower on AWS (ROSA) and non-STS clusters)
```

File: content/o11y/openshift-logging-lokistack/index.md [hunk 7]
```diff
@@ -0,0 +1,572 @@
...
+        ```bash
+        logging-view-plugin-bd5978d6d-9sc5v            1/1     Running   0          8m41s
+        ```  
+
+    1. Confirm you can see the Logging section of the console under the Observe tab:
+
+        ![logging in console](./logging-in-console.png)
+
+
+1. At this point OpenShift logging is installed and configured and is ready to receive logs.
```
Comment (bhardesty) [inline content/o11y/openshift-logging-lokistack/index.md:444]:
```suggestion
At this point OpenShift logging is installed and configured and is ready to receive logs.
```

**Review by bhardesty — COMMENTED** [review]
This looks good, and the procedures/steps flow well. I didn't necessarily review for style, but I did note a few small things that I think might improve the readability/flow.

**Review by paulczar — APPROVED** [review]

**Pull request merged**